src/Dapr.AzureFunctions.Extension/Bindings/DaprSecretConverter.cs

```diff
@@ -38,7 +38,8 @@ public DaprSecretConverter(DaprServiceClient daprClient)
             CancellationToken cancellationToken)
         {
             JObject result = await this.GetSecretsAsync(input, cancellationToken);
-            return result.ToObject<Dictionary<string, string>>();
+            var obj = result.ToObject<Dictionary<string, string>>();
```

Comment:
This may result in null reference exception, can we have null check here?

Reply:
Can you add the code suggestion here so I can just apply it? Using the Github suggestion feature.

Reply:
`result` is not nullable, but the `GetSecretsAsync` can throw an exception. Using #87 to track it.